generic_routines.sh

```diff
@@ -89,40 +90,42 @@ function capitalize_first_letter {
 ## -GEOD-/-ERAD-/-ENAD- are loaded as public from now on
 peach_api_privacy_status() {
     expAcc=$1
-    exp_import=$(echo $expAcc | awk -F"-" '{print $2}')
-
-    if [ $exp_import == "MTAB" ]; then
-        response=$(curl -s "${PEACH_API_URI}/privacy.txt?acc=$expAcc")
-        if [ -z "$response" ]; then
-            die "WARNING: Got empty response from ${PEACH_API_URI}/privacy.txt?acc=$expAcc" 0
-        fi
-        privacyStatus=$(echo $response | awk '{print $2}' | awk -F":" '{print $2}')
-        ## if not MTAB, ie. GEOD or ENAD or ERAD are all loaded as public
-    else
-        privacyStatus=$(echo "public")
-    fi
-
-    echo $privacyStatus
+    #exp_import=$(echo $expAcc | awk -F"-" '{print $2}')
+    #
+    #if [ $exp_import == "MTAB" ]; then
+    #    response=$(curl -s "${PEACH_API_URI}/privacy.txt?acc=$expAcc")
+    #    if [ -z "$response" ]; then
+    #        die "WARNING: Got empty response from ${PEACH_API_URI}/privacy.txt?acc=$expAcc" 0
+    #    fi
+    #    privacyStatus=$(echo $response | awk '{print $2}' | awk -F":" '{print $2}')
+    #    ## if not MTAB, ie. GEOD or ENAD or ERAD are all loaded as public
+    #else
+    #    privacyStatus=$(echo "public")
+    #fi
+    #
+    #echo $privacyStatus
+    echo " peach_api_privacy_status function is deprecated "
```

Comment:
Maybe "Privacy status can no longer be obtained via the Peach API" so it's still an informative message even if the user has no knowledge of the function.